scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -26,9 +26,23 @@ namespace <%= namespace %> {
             }
         }
 
-        private CartLineItems _LineItems;
+        private bool IsSaved {
+            get {
+                return IsCreated && LineItems.IsSaved;
+            }
+        }
 
+        private bool IsCreated {
+            get {
+                return CurrentCheckout != null;
+            }
+        }
+
+        private const float POLL_DELAY_SECONDS = 0.5f;
+        private CartLineItems _LineItems;
+        private List<String> DeletedLineItems = new List<string>();
         private ShopifyClient Client;
+        private Checkout CurrentCheckout = null;
```

Comment:
I use rider which uses Resharper under the hood, but i dont think it runs as a standalone thing yet.... 

I can always just do a one-off pr that just brings it into rider and runs a lint on the whole thing... we would have to make sure theres no other work going on that would conflict